tox.ini

```diff
@@ -25,6 +25,7 @@ passenv =
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
+    -r{toxinidir}/requirements.txt
```

Comment:
Out of curiosity, why do we need these deps?

Reply:
Mis-added the deps, my bad! Removed :)